orion/pkg/controllers/session_controller.go

```diff
@@ -0,0 +1,85 @@
+package controllers
+
+import (
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/services"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func GetAllSessionsByClassId(c *gin.Context) {
+	classId := c.Param("classId")
+
+	sessionList, err := services.SessionService.GetAllByClassId(classId)
+	if err != nil {
+		c.String(http.StatusInternalServerError, err.Error())
```

Comment:
From a previous PR that I made last week. Make sure that before you use c.String(...), use c.Error(err) so the error will be displayed on the webserver side.